specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/network/group.sasl.f1.j5.s3.l3.h3.handshake.v1/leader/client.rpt

```diff
@@ -176,7 +176,7 @@ read 20                                 # size
      -1s                                # authentication bytes
      0L                                 # session lifetime
 
-write 119                               # size
+write 82                               # size
```

Comment:
Comment has vertical misalignment
(also for other similar scripts with `${instanceId}` replaced by `"zilla"`)